app/src/main/java/com/example/studentcoursebooking_seg2105_group6/MainActivity.java

```diff
@@ -26,6 +32,36 @@ public class MainActivity extends AppCompatActivity {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        //function find id of button on screen
```

Comment:
I understand what you're trying to do here (Line 47). But the user sign in/registration has to be validated against the user database instead of in-memory (i.e., in the app using if-else conditions).

Reply:
For example, the admin creds are already stored in db. ![image](https://user-images.githubusercontent.com/77720215/193483381-a502883a-c473-49a8-a550-76088141373b.png)